docs/guides/medical.md

```diff
@@ -0,0 +1,181 @@
+Pamiętaj, o ile nie musisz, nie obciążaj medyka w trudnych sytuacjach. Jeśli jesteś przytomny to najpewniej możesz opatrzyć się sam. Korzystaj z pomocy medyka kiedy wiesz, że ten akurat nie zajmuje się nieprzytomnymi. 
+
+Cechy staz taktycznych:
+
+- zakładane tylko na ręce i nogi,
+- drastycznie szybsze w użyciu od bandażowania (kupują czas medykowi na opatrzenie torsu i głowy, piechocie pozwalają na nieprzerwane prowadzenie ognia w przypadku obrażeń kończyn),
+- wielokrotnego użytku - po ściągnięciu wracają do ekwipunku ściągającego,
+- po kilku minutach od założenia powodują narastający ból,
+- wlewy medyczne (krew, morfina) nie przynoszą efektu przy podaniu drogą zastazowanej kończyny do czasu jej zdjęcia.
+
+`++` = Super | `+` = Zwiększona | Brak = Standardowa
+
+| Rana\Bandaż	| Jałowy			|Elastyczny	|Uciskowy	| QuikClot		|  Pol\Eng |
+|:-------------:|:-----------------:|:---------:|:---------:|:-------------:|:--------:|
+| Draśnięcie	| ++				| 			| 			| 				| Scrape |
+| Płatowa		|					| ++ 		| 			| 				| Avulsion |
+| Stłuczenie	|	n/d				| n/d 		| n/d		| n/d			| Bruise |
+| Zgniecenie	| 					| ++ 		| 			| 				| Crushed |
+| Cięta			| ++ 				| 			| 			| 				| Cut |
+| Rozerwana		| ++ 				| 			| 			| 				| Tear |
+| Postrzałowa	| + 				| + 		| ++ 		| ++ 			| Velocity |
+| Kłuta			| + 				| + 		| ++ 		| ++ 			| Puncture |
+| **Pol\Eng**	|**Field Dressing**	|**Elastic**|**Packing**|**QuikClot**	|**Dressing\Wound**|
+
+Efektywności bandaży (w plusach) można odszyfrować zgodnie z poniższą tabelą, zawierającą:
+
+- czas, po którym może otworzyć się rana (przedział od wartości minimalnej do maksymalnej)
+- szansę, że otworzy się w tym czasie (jeśli się nie otworzy w tym czasie to nigdy się nie otworzy)
+- kolejno ile ran danej wielkości można załatać jednym bandażem
+
+| Efektywność | | + | ++ |
+|-----------------------|:----:|:----:|:-----:|
+| Czas otwierania (min) | 5-10 | 6-20 | 10-30 |
+| Szansa otwarcia | 0.4 | 0.3 | 0.1 |
+| Ef. Duże | 2 | 4 | 8 |
+| Ef. Średnie | 3 | 5 | 9 |
+| Ef. Małe | 4 | 6 | 10 |
+
+**Podsumowując:**
+- Rodzaj bandaża nie odgrywa takiej roli. Jeśli masz dostęp do szycia możesz bandażować się dowolnym bandażem.
+- Jeżeli sytuacja nie wymaga natychmiastowej pomocy medyka, nie wołaj go. Użyj staz, zabandażuj się, wtedy zapytaj medyka czy może Ci pomóc.
+
+
+## Krew
+
+Podczas rozgrywki krew jest jednym z najcenniejszych zapasów. Z tego powodu, w miarę możliwości trzymaj przy sobie krew i uzupełniaj jej zapas, jeżeli jest taka możliwość. 0,5 - 1,0 l w zupełności wystarczy, gdybyś stracił przytomność oraz znacząco odciążysz medyka. Jeżeli jesteś przytomny, nie potrzebujesz przetoczenia krwi do momentu pojawienia się szarego ekranu.
```

Comment:
`0,5 - 1,0 l w zupełności wystarczy, gdybyś stracił przytomność oraz znacząco odciążysz medyka.` this seems weird, try splitting that into 2 sentences.

Maybe try:
`0,5 - 1,0 l w twoim ekwipunku pozwoli znacznie odciążyć medyka a przy okazji może Cię również uratować, gdy zapasy medyków będą na wyczerpaniu.`